src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -133,11 +133,6 @@ export default class GeneSetSelector extends QueryStoreComponent<{}, {}> {
                             <i className={'fa fa-external-link'} />
                         </a>
                     }
-                    promises={[
```

Comment:
don't want to show loader anymore.  it's pointless and distracting to typing user